Gemfile.lock

```diff
@@ -104,13 +104,13 @@ PLATFORMS
 
 DEPENDENCIES
   appraisal (= 1.0.3)
-  bundler
+  bundler (~> 1.11)
   chronic
   git_reflow!
   pry-byebug
-  rake
+  rake (~> 10.0)
   rdoc
-  rspec (~> 3.0.0)
+  rspec (~> 3.0)
```

Comment:
Note, I tried updating rspec to 3.4, and it fails the spec suite miserably.  Going to require some effort to fix compatibility with rspec 3.4.